Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/Shared/_LayoutNoHeaderAndFooter.cshtml

```diff
@@ -28,31 +31,52 @@
 
    <title>@ViewData["Title"] - Manage an academy conversion</title>
    <link rel="stylesheet" href="~/dist/site.css"/>
-   @if (showAnalytics)
-   {
-      <!-- Google Tag Manager -->
-      <script asp-add-nonce>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-		new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-		j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-		'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-		})(window,document,'script','dataLayer','GTM-5H6G773');</script>
-      <!-- End Google Tag Manager -->
-   }
+
+   @if (enableGoogleAnalytics)
+    {
+        <!-- Google Tag Manager -->
+        <script asp-add-nonce>
+            (function (w, d, s, l, i) {
+                w[l] = w[l] || []; w[l].push({
+                    'gtm.start':
+                        new Date().getTime(), event: 'gtm.js'
+                }); var f = d.getElementsByTagName(s)[0],
+                    j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
+                        'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
+            })(window, document, 'script', 'dataLayer', 'GTM-5H6G773');
+         </script>
+         <!-- End Google Tag Manager -->
+    }
+
+    @if (enableAppInsightsAnalytics)
+    {
+      <!-- Application insights -->
+      <script type="text/javascript" integrity="sha384-g/ZkzetdQypWdY0NBZT5r2L3BR9/hURD8OBcd1rEaBpgX6QC7EaTL+o+mzWrBcXW" crossorigin="anonymous" src="https://js.monitor.azure.com/scripts/b/ext/ai.clck.2.8.18.min.js"></script>
```

Comment:
This is a slight concern, this will expose 
 connectionString: '@Configuration["ApplicationInsights:ConnectionString"]',
            authenticatedUserId: '@authenticatedUserId'
            
            to the user. Is that permissable?

Reply:
yeah its just a push only key just as non-sensitive as the Google analytics ID :)